apps/hubble-stats/data/getOverviewChipsData.ts

```diff
@@ -1,14 +1,59 @@
-import { randNumber } from '@ngneat/falso';
+import { formatEther } from 'viem';
+import vAnchorClient from '@webb-tools/vanchor-client';
+
+import { vAnchorAddresses, allSubgraphUrls } from '../constants';
 
 type OverviewChipsDataType = {
-  tvl: number;
-  volume: number;
+  tvl: number | undefined;
+  volume: number | undefined;
 };
 
 export default async function getOverviewChipsData(): Promise<OverviewChipsDataType> {
-  await new Promise((r) => setTimeout(r, 1000));
+  let tvl, volume;
+
+  try {
+    const tvlVAnchorsByChainsData =
+      await vAnchorClient.TotalValueLocked.GetVAnchorsTotalValueLockedByChains(
+        allSubgraphUrls,
+        vAnchorAddresses
+      );
+
+    tvl = tvlVAnchorsByChainsData.reduce((tvlTotal, vAnchorsByChain) => {
+      const tvlVAnchorsByChain = vAnchorsByChain.reduce(
+        (tvlTotalByChain, vAnchor) =>
+          tvlTotalByChain + +formatEther(BigInt(vAnchor.totalValueLocked)),
+        0
+      );
+      return tvlTotal + tvlVAnchorsByChain;
+    }, 0);
+  } catch {
+    tvl = undefined;
+  }
+
+  try {
+    const depositVAnchorsByChainsData =
+      await vAnchorClient.Deposit.GetVAnchorsDepositByChains(
+        allSubgraphUrls,
+        vAnchorAddresses
+      );
+
+    volume = depositVAnchorsByChainsData.reduce(
```

Comment:
`depositVAnchorsByChainsData` can be `null` here.

Reply:
Same as [this](https://github.com/webb-tools/webb-dapp/pull/1521#discussion_r1292918551)